src/Application.vala

```diff
@@ -26,11 +26,20 @@ public class Installer.App : Gtk.Application {
 
     public static bool test_mode;
 
+    private static Installer.App instance = null;
```

Comment:
This and 39–42 are only needed for the pop-specific quit https://github.com/elementary/installer/pull/335/files#r195868345